### ISSUE TITLE
Some missed editorial changes from the Dec 2024 WCAG publication

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag#4427]
"Web" is still used instead of "web" in a few places. 
- [ ] definition of [accessibility supported](https://www.w3.org/TR/WCAG22/#dfn-accessibility-supported) - Note 5, last sentence
- [ ] definition of [role](https://www.w3.org/TR/WCAG22/#dfn-role) still has "Web"
- [ ] definition of [technology (web content)](https://www.w3.org/TR/WCAG22/#dfn-technologies) - Note 2, second from last word.
- [ ] Privacy Considerations, last sentence of the paragraph after "This section is non-normative." - "Success criteria" should be "success criteria"

**Comment by patrickhlauke** [comment]
Filed https://github.com/w3c/wcag/pull/4428
